postcss.config.js

```diff
@@ -0,0 +1,12 @@
+const colors = require('./src/config/colors')
+
+module.exports = {
+  plugins: [
+    require('postcss-simple-vars')({
+      variables: function () {
+        return colors
+      }
```

Comment:
Для таких безымянных функций колбеков, обычно использую стрелочные функции, так код выглядит лаконичнее)

```suggestion
      variables: () => colors
```